sound/soc/intel/boards/sof_rt5682.c

```diff
@@ -207,6 +207,13 @@ static int sof_rt5682_codec_init(struct snd_soc_pcm_runtime *rtd)
 	return ret;
 };
 
+static void sof_rt5682_codec_exit(struct snd_soc_pcm_runtime *rtd)
```

Comment:
looks good @fredoh9 but the commit title is wrong. This should not be a fixup!, we've already provided this upstream. Only add the fixup! whe the code is still queued in topic/sof-dev-rebase.

Do you mind changing this? It'll help maintain the code and send it upstream. Thanks!

Reply:
Sure! I will update commit title and update the branch shortly